packages/@aws-cdk/toolkit/lib/toolkit/toolkit.ts

```diff
@@ -647,13 +640,10 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
    */
   private async _rollback(assembly: StackAssembly, action: 'rollback' | 'deploy' | 'watch', options: RollbackOptions): Promise<void> {
     const ioHost = withAction(this.ioHost, action);
-    const timer = Timer.start();
+    const synthTimer = Timer.start();
     const stacks = assembly.selectStacksV2(options.stacks);
     await this.validateStacksMetadata(stacks, ioHost);
-    const synthTime = timer.end();
-    await ioHost.notify(info(`\n✨  Synthesis time: ${synthTime.asSec}s\n`, 'CDK_TOOLKIT_I5001', {
```

Comment:
This changes to code to `CDK_TOOLKIT_I0000` which is the universal code for synth time.